client/my-sites/checkout/cart/style.scss

```diff
@@ -430,6 +439,29 @@ div.popover-cart__popover {
 	.jetpack-logo {
 		margin: 32px 0;
 	}
+
+}
+
+
+.guarantee-seal {
+
+	img {
+		width: 40%;
+		transform: rotate( 15deg );
+	}
+
+	@include breakpoint( '<660px' ) {
+		margin: 20px auto;
+		text-align: center;
```

Comment:
This center aligns for mobile but not for desktop. I've [pushed](https://github.com/Automattic/wp-calypso/pull/34677/commits/546a1c8562edb08e3cd46053793cffab38e0c6af) in a small fix to fix this.

Reply:
Yes, it was intentional to not align center on desktop since it's left-aligned in its column, but thanks anyway!